tox.ini

```diff
@@ -0,0 +1,88 @@
+[tox]
+envlist =
+    py{39,310,311}-test{,-cov}
+    linkcheck
+    codestyle
+    pep517
+    securityaudit
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+isolated_build = true
+
+[testenv]
+# Suppress display of matplotlib plots generated during docs build
+setenv =
+    MPLBACKEND=agg
+
+# Pass through the following environment variables which may be needed for the CI
+passenv = HOME,WINDIR,LC_ALL,LC_CTYPE,CC,CI
+
+# Run the tests in a temporary directory to make sure that we don't import
+# this package from the source tree
+changedir = .tmp/{envname}
+
+# tox environments are constructed with so-called 'factors' (or terms)
+# separated by hyphens, e.g. test-devdeps-cov. Lines below starting with factor:
+# will only take effect if that factor is included in the environment name. To
+# see a list of example environments that can be run, along with a description,
+# run:
+#
+#     tox -l -v
+#
+description =
+    run tests
+    cov: and test coverage
+
+# The following provides some specific pinnings for key packages
+deps =
+    # NOTE: Add/remove as needed
+
+# The following indicates which extras_require from setup.cfg will be installed
+extras =
+    test
+    # Uncomment when we have all again in setup.cfg
+    #alldeps: all
+
+commands =
+    pip freeze
+    !cov: pytest --pyargs pypolyclip {toxinidir}/docs {posargs}
+    cov: pytest --pyargs pypolyclip {toxinidir}/docs --cov pypolyclip --cov-config={toxinidir}/setup.cfg {posargs}
+    cov: coverage xml -o {toxinidir}/coverage.xml
+
+pip_pre =
+    predeps: true
+    !predeps: false
+
+[testenv:linkcheck]
+changedir = docs
+description = check the links in the HTML docs
+extras = docs
+commands =
+    pip freeze
+    sphinx-build -W -b linkcheck . _build/html
+
+[testenv:codestyle]
+skip_install = true
+changedir = .
+description = check code style, e.g. with flake8
+deps = flake8
+commands = flake8 pypolyclip --count
+
+[testenv:securityaudit]
+skip_install = true
+changedir = .
+description = security audit with bandit
+deps = bandit
+commands = bandit -r pypolyclip -c .bandit.yaml
```

Comment:
This test will fail because there is no `.bandit.yaml` file in the repo.  Please add one.